server/plugins/schema-plugin-ob-mysql/src/main/java/com/oceanbase/odc/plugin/schema/obmysql/parser/GetDBTableByParser.java

```diff
@@ -47,4 +48,6 @@ public interface GetDBTableByParser {
      * Get table partition info by parse table ddl
      */
     DBTablePartition getPartition();
+
+    List<DBColumnGroupElement> getColumnGroups();
```

Comment:
I have a question: Why are the column groups of the index placed on the `db-browser` layer, and the column groups in the table ddl are placed on the `GetDBTableByPaser` layer?

Reply:
The `columnGroups` in db-brower is for service layer. We place it in `GetDBTableByParser` is because the column group info could only be parsed from ddl. There is no system views for it.😂